### DISCUSSION TITLE
https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/echarts.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#620] [Announcements]
# https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/echarts.html

让你 VuePress 站点中的 Markdown 文件支持图表。 此插件使用 ECharts (https://echarts.apache.org/zh/index.html) 提供相应功能。 配置 tab TS tab JS 格式 使用 JSON 如果你可以很轻松的生成数据，你可以直接通过一个 JSON 代码块来提供 Echarts 配置: 使用...

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/echarts.html

**Comment by song-jun** [comment]
使用 await 和 fetch 来从网络请求中获取数据。
如何使用

**Comment by song-jun** [comment]
从网络请求中获取数据
内置axios吗
